README.md

```diff
@@ -1 +1,67 @@
+| RLGL                                   | Nithin Steven F | A classic game of red light greenlight made with wonderland engine.                                                                                                                                                                                                                                                                                                                      | https://heyvr.io/arcade/games/redlight-greenlight        | No          | N/A                                            |
+| tvroom                                 | Nithin Steven F | Multiplayer space where user can host and join rooms with friends (upto 5 users) and watch immersive web content together.                                                                                                                                                                                                                                                               | tvroom.in                                                | No          | N/A                                            |
+| mr NOT - enough IS enough              | Pipo            | My dearest puppet, did you miss me? I did. Why don't you STAND UP, SQUEEZE those buttons, and get ready to THROW? I am here for you, don't make me wait.                                                                                                                                                                                                                                 | https://heyvr.io/arcade/games/enough-is-enough           | Yes         | https://github.com/signorpipo/enough-is-enough |
+| Mescolato                              | Pipo            | clear your mind                                                                                                                                                                                                                                                                                                                                                                          | https://signorpipo.itch.io/mescolato                     | Yes         | https://github.com/signorpipo/mescolato        |
+| De-Evil-ery                            | Elia & Marco    | That stupid Prince Ugo! He’s trying to wake up the Princess after you finally trapped her in the mystic blue sphere! Don’t panic! De-Evil-ery is the company that can support you in your fights! Destroy the mecha-birds sent by Prince Ugo to wake up the Princess, and gain evilness points to buy new weapons! Be Careful though! If the true love meter fills up… IT’S GAME OVER!!! | https://signorpipo.itch.io/de-evil-ery                   | Yes         | https://github.com/signorpipo/devilery         |
```

Comment:
```suggestion
| De-Evil-ery                            | Pipo & Marco    | That stupid Prince Ugo! He’s trying to wake up the Princess after you finally trapped her in the mystic blue sphere! Don’t panic! De-Evil-ery is the company that can support you in your fights! Destroy the mecha-birds sent by Prince Ugo to wake up the Princess, and gain evilness points to buy new weapons! Be Careful though! If the true love meter fills up… IT’S GAME OVER!!! | https://signorpipo.itch.io/de-evil-ery                   | Yes         | https://github.com/signorpipo/devilery         |
```

To be consistent. I wonder if it would be common on awesome-lists to link the authors?